src/Simulation.C

```diff
@@ -51,6 +51,11 @@ Simulation::Simulation(const YAML::Node& root_node)
   cudaDeviceGetLimit(&default_stack_size, cudaLimitStackSize);
   cudaDeviceSetLimit(cudaLimitStackSize, nalu_stack_size);
 #endif
+
+#ifdef KOKKOS_ENABLE_HIP
```

Comment:
Are these stale or place holders for the future?

Reply:
yes